wLabs/LabSetup.md

```diff
@@ -0,0 +1,595 @@
+<td></td>
+</tr>
+<tr>
+<td>Interject_LocalTimeZoneOffset</td>
+<td>System Parameter</td>
+<td></td>
+<td></td>
+</tr>
+</tbody>
+</table>
+</div>
+
+<button class="collapsible-parameter">**NorthwindFixed**<br>Stored Procedure: [demo].[Northwind_Customers_FixedPull]</button>
+<div markdown="1" class="panel-parameter">
+<table>
+<tbody>
+
+<td class="pph"><b>Parameter</b></td>
+<td class="pph"><b>Parameter Type</b></td>
+<td class="pph"><b>Data Type</b></td>
+<td class="pph"><b>Direction</b></td>
+
+<tr>
+<td>Interject_NTLogin</td>
+<td>System Parameter</td>
+<td></td>
+<td></td>
+</tr>
+<tr>
+<td>Interject_ReturnError</td>
+<td>System Parameter</td>
+<td></td>
+<td></td>
+</tr>
+</tbody>
+</table>
+</div>
+
+<button class="collapsible-parameter">**NorthwindInvoiceSave**<br>Stored Procedure: [demo].[Northwind_Invoices_Save]</button>
+<div markdown="1" class="panel-parameter">
+<table>
+<tbody>
+
+<td class="pph"><b>Parameter</b></td>
+<td class="pph"><b>Parameter Type</b></td>
+<td class="pph"><b>Data Type</b></td>
+<td class="pph"><b>Direction</b></td>
+
+<tr>
+<td>Interject_ReturnError</td>
+<td>System Parameter</td>
+<td></td>
+<td></td>
+</tr>
+<tr>
+<td>Interject_RequestContext</td>
+<td>System Parameter</td>
+<td></td>
+<td></td>
+</tr>
+
+</tbody>
+</table>
+</div>
+<button class="collapsible-parameter">**NorthwindSalesPull**<br>Stored Procedure: [demo].[Northwind_Customers_RangeDetailPull]</button>
+<div markdown="1" class="panel-parameter">
+<table>
+<tbody>
+
+<td class="pph"><b>Parameter</b></td>
+<td class="pph"><b>Parameter Type</b></td>
+<td class="pph"><b>Data Type</b></td>
+<td class="pph"><b>Direction</b></td>
+
+<tr>
+<td>UnitsInStockMin</td>
+<td>Formula Parameter</td>
+<td>varchar</td>
+<td>input</td>
+</tr>
+<tr>
+<td>Category</td>
+<td>Formula Parameter</td>
+<td>varchar</td>
+<td>input</td>
+</tr>
+<tr>
+<td>Interject_NTLogin</td>
+<td>System Parameter</td>
+<td></td>
+<td></td>
+</tr>
+<tr>
+<td>Interject_ReturnError</td>
+<td>System Parameter</td>
+<td></td>
+<td></td>
+</tr>
+</tbody>
+</table>
+</div>
+
+### Step 4: Setting up FinCube
+
+[FinCube](/wIndex/FinCube---The-Financial-Cube.html) is a special Data Portal that is used for financial reporting. This Portal is called "Demo_Interject_FinCube" in the Lab files and is used in 6 labs:
+
+* Lab 1.3 Financial Report
+* Lab 2.4 Financial Report
+* Lab 4.3 Financial Report
+* Lab 5.5 Using the Retain Feature
+* Lab 7.2 Basic Distribution
+* Lab 7.3 Advanced Distribution
+
+Due to the complexities of this Data Portal, it is advised to use the API connection for this Data Portal. The following steps through how to set up this Portal.
+
+#### Setting up the FinCube Data Connection
+
+The FinCube Portal can use an API connection. You can use the following settings to set up the Data Connection:
+
+* **Name:** \<Your own custom Name>
+* **Description:** \<Your own custom Description>
+* **Connection Type:** Web Api
+* **Api Root Url:** https://data-api.gointerject.com/api
```

Comment:
does this mean we are letting anyone run read\write requests on this data api? Seems abit open for an api that could be accessed by anyone who follows this walkthrough. maybe